src/enums.h

```diff
@@ -47,9 +47,13 @@ typedef enum {
     STORE_PENDING
 } store_status_t;
 
+///Store entry swapping out states.
 typedef enum {
-    SWAPOUT_NONE,
-    SWAPOUT_WRITING,
+    SWAPOUT_NONE, ///< the store entry is in a 'disconnected' from disk state
```

Comment:
`disconnected` can be easily misinterpreted as "was connected but not longer connected". How about "StoreEntry is currently not associated with any disk store entry". And, to make it more symmetric with other definitions:

```
/// StoreEntry is currently not associated with any disk store entry.
/// Does not guarantee (or preclude!) a matching disk store entry existence.
```

Reply:
Done at  c0a470d.